### ISSUE TITLE
feat: 多値分類モデル（LGBMMultiClassifier）の実装

### ISSUE THREAD

**Issue by maruyamamtk** [maruyamamtk/keiba_prediction#378]
## 概要
LightGBM `multiclass` objective を使った着順分類モデルを新規実装する。
着順を5クラスに分類し、各クラスの確率から複勝率を計算する。

## クラス定義

| クラスラベル | 対象 | 意味 |
|------------|------|------|
| 0 | 1着 | 1位 |
| 1 | 2着 | 2位 |
| 2 | 3着 | 3位 |
| 3 | 4〜10着 | 中位 |
| 4 | 11着以降 | 下位（非掲示板） |

### 新規ファイル: `src/models/lgbm_multiclassifier.py`
- `LGBMMultiClassifierConfig`: objective=softmax, num_class=5, metric=multi_logloss
- `LGBMMultiClassifier(LGBMModelBase)`
  - `train(X_train, y_train, X_valid, y_valid)`: ラベルはfinish_position→クラスへの変換済み整数
  - `predict_proba(X) -> np.ndarray`: shape [n_horses, 5] のクラス確率行列
  - `predict_place_prob(X) -> np.ndarray`: P(class=0) + P(class=1) + P(class=2) = 複勝率（0〜1）
  - `save() / load()`: meta.jsonに `model_type=multiclassifier` を記録

### ユーティリティ関数: `src/models/lgbm_multiclassifier.py` 内
```python
def finish_position_to_class(finish_position: int) -> int:
    """着順→クラスラベル変換"""
    if finish_position == 1: return 0
    if finish_position == 2: return 1
    if finish_position == 3: return 2
    if finish_position <= 10: return 3
    return 4
```

### テスト: `tests/test_lgbm_multiclassifier.py`
- クラスラベル変換のテスト
- train→predict_probaの形状確認
- predict_place_probが[0,1]に収まることを確認
- save/loadの往復テスト

## 完了条件
- [ ] LGBMMultiClassifier が学習・推論・保存・読み込みできる
- [ ] predict_place_prob() の出力が各馬0〜1、レース内合計が約3になる
- [ ] テスト全件通過

## 依存
- issue #377（不要モデル削除）完了後に実装

**Comment by maruyamamtk** [comment]
誤って起票。issue #358実装済みのLGBMRankerMultiを残すため、新規LGBMMultiClassifier実装は不要。
